doc/tools/makerst.py

```diff
@@ -1004,6 +1009,16 @@ def make_heading(title, underline):  # type: (str, str) -> str
     return title + "\n" + (underline * len(title)) + "\n\n"
 
 
+def make_footer():
+    # Generate reusable abbreviation substitutions.
+    # This way, we avoid bloating the generated rST with duplicate abbreviations.
+    return (
+        ".. |virtual| replace:: :abbr:`virtual (This method should typically be overridden by the user to have any effect.)`\n"
+        ".. |const| replace:: :abbr:`const (This method has no side effects. It doesn't modify any of the instance's member variables.)`\n"
+        ".. |vararg| replace:: :abbr:`vararg (This method accepts any number of arguments after the ones described here.)`"
+    )
```

Comment:
`black` has *opinions*:
https://github.com/godotengine/godot/pull/40872/checks?check_run_id=927477086

Reply:
Strangely enough, it didn't make any changes when I ran it locally with `-l120`.

Reply:
We use black from their Git `master` branch, which seems to have gained more capabilities to reflow long strings. See #40731.

Reply:
I wrapped the block with `# fmt: off` and `# fmt: on` to work around this.